db/redis.js

```diff
@@ -116,8 +116,17 @@ async function addVitals(locationid, signalStrength, cloudDisconnects) {
   client.xadd(`vitals:${locationid}`, 'MAXLEN', '~', '10000', '*', 'strength', signalStrength, 'cloudDisc', cloudDisconnects)
 }
 
-async function addEdgeDeviceHeartbeat(locationid, doorStatus, doorTime, insTime) {
-  client.xadd(`heartbeat:${locationid}`, 'MAXLEN', '~', '10000', '*', 'doorStatus', doorStatus, 'doorTime', doorTime, 'insTime', insTime)
+// ignore comments included to allow arguments to be split across lines in pairs
+// prettier-ignore
+/* eslint-disable function-call-argument-newline */
+async function addEdgeDeviceHeartbeat(locationid, missedDoorMessages, doorLowBattery, doorHeartbeatReceived, stateTransitions) {
+  client.xadd(
+    `heartbeat:${locationid}`, 'MAXLEN', '~', '10000', '*',
+    'missedDoorMessages', missedDoorMessages,
+    'doorLowBattery', doorLowBattery,
+    'doorHeartbeatReceived', doorHeartbeatReceived,
+    'stateTransitions', stateTransitions
```

Comment:
It would be great if you could htink about all these names a little more to see if you can make it clearer what types of values we would expect from them. For example, this one might be better as `millisSinceDoorHeatbeat`.